package.json

```diff
@@ -21,6 +21,20 @@
     "url": "https://github.com/github/hubot.git"
   },
   "dependencies": {
+<<<<<<< HEAD
```

Comment:
Looks like you've got a conflict here!
Not sure if you meant to change the package.json file, if not just copy the latest one from [here]( https://github.com/shulance/gir/blob/master/package.json) and paste it into your local version. Do a push and then we should see the pull request update.
Give me a shout if you need a hand!